packages/xo-server/src/api/disk.mjs

```diff
@@ -79,9 +79,9 @@ create.resolve = {
 
 // -------------------------------------------------------------------
 
-async function handleExportContent(req, res, { xapi, id }) {
-  const stream = await xapi.exportVdiContent(id)
-  req.on('close', () => stream.cancel())
+async function handleExportContent(req, res, { xapi, id, format = 'vhd' }) {
+  const stream = format === 'vmdk' ? await xapi.exportVdiAsVMDK(id) : await xapi.exportVdiContent(id)
+  req.on('close', () => stream.destroy())
```

Comment:
Is this fix valid even without this PR?

Reply:
no, it's because stream was an HTTP Result but it is now a real ReadStream.